docs/source/Changelog.rst

```diff
@@ -1,6 +1,17 @@
 Changelog
 =========
 
+2.7.0
+------
+
+- Add support for django `3.2`.
+- Allow blocking users/emails from adding/reacting with comments.
+- Improve commenting anonymously UI.
+- Enhance API docs by adding openapi and swagger page to RTD.
```

Comment:
i am not a big fan of using acronyms without defining it at some place. Not everyone may get the context at all times.
Maybe we can add the link to the `readthedocs` page here.

Reply:
i was originally talking about the RTD acronym used here. But, may this clean-up can be done in a separate one, as this will block the release.